<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="5" hostname="DMS" name="com.Pages.Tests.TestBase" tests="6" failures="0" timestamp="17 May 2017 15:44:52 GMT" time="0.009" errors="1">
  <testcase name="TC_001" time="0.002" classname="com.Pages.Tests.TestBase">
    <skipped/>
  </testcase> <!-- TC_001 -->
  <testcase name="TC_006" time="0.002" classname="com.Pages.Tests.TestBase">
    <skipped/>
  </testcase> <!-- TC_006 -->
  <testcase name="TC_003" time="0.001" classname="com.Pages.Tests.TestBase">
    <skipped/>
  </testcase> <!-- TC_003 -->
  <testcase name="TC_005" time="0.002" classname="com.Pages.Tests.TestBase">
    <skipped/>
  </testcase> <!-- TC_005 -->
  <testcase name="Setup" time="0.000" classname="com.Pages.Tests.TestBase">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.Pages.Tests.TestBase.Setup(TestBase.java:49)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:216)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:143)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:169)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:753)
	at org.testng.TestRunner.run(TestRunner.java:607)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
	at org.testng.TestNG.runSuites(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1096)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </error>
  </testcase> <!-- Setup -->
  <testcase name="TC_004" time="0.002" classname="com.Pages.Tests.TestBase">
    <skipped/>
  </testcase> <!-- TC_004 -->
</testsuite> <!-- com.Pages.Tests.TestBase -->
